modules/custom/az_finder/config/quickstart/views.view.az_events.yml

```
@@ -1371,6 +1371,326 @@ display:
         - 'user.node_grants:view'
         - user.permissions
       tags: {  }
+  block_1:
+    id: block_1
```

It looks like the original view display has a different ID:
```suggestion
  az_past_row:
    id: az_past_row
```